test/e2e/pages/vl-accordion.page.js

```diff
@@ -20,18 +20,26 @@ class VlAccordionPage extends Page {
         return this._getAccordion('#accordion-javascript-toggle');
     }
 
-    async getJSAccordionOpenButton() {
-        return new VlButton(this.driver, '#open-accordion');
+    async clickJSAccordionOpenButton() {
+        return await this._clickJSAccordionButton('#open-accordion');
     }
 
-    async getJSAccordionCloseButton() {
-        return new VlButton(this.driver, '#close-accordion');
+    async clickJSAccordionCloseButton() {
+        return await this._clickJSAccordionButton('#close-accordion');
     }
 
-    async getJSAccordionToggleButton() {
-        return new VlButton(this.driver, '#toggle-accordion');
+    async clickJSAccordionToggleButton() {
+        return await this._clickJSAccordionButton('#toggle-accordion');
     }
 
+  async _clickJSAccordionButton(selector) {
+      const button = await new VlButton(this.driver, selector);
+      await this.driver.executeScript('return arguments[0].scrollIntoView()',
```

Comment:
Ik zou dit precies wel extracten en misschien zelfs ineens in de core VlElement steken.
Of zouden we de VlElement op die manier te fel bloaten, @coemans en @pascalriquier ?

Reply:
Als het geen kwaad kan zou ik het inderdaad op elke click toepassen door de click te overschrijven in VlElement.

Reply:
evt een scrollIntoView functie op de vl-element zetten maar zo ver als @SanderKleykens zou ik niet gaan

Reply:
Mijn comment ging over de regel met de scrollIntoView, niet over de hele click :)